lib/htmlparser-to-vdom.js

```diff
@@ -70,25 +80,32 @@ module.exports = function createConverter (VNode, VText) {
             var key;
 
             var attributes = {
-                dataset: dataset
+                dataset: dataset,
+                attributes : {}
             };
 
             Object.keys(tag.attribs).forEach(function (name) {
                 var value = tag.attribs[name];
 
+                var attr = name;
+
                 if (attributesToRename[name]) {
-                    attributes[attributesToRename[name]] = value;
-                    return;
+                    attr = attributesToRename[name];
                 }
                 if (attributesToDecode[name]) {
-                    attributes[name] = decode(value);
-                    return;
+                    value = decode(value);
                 }
                 if (name === 'style') {
-                    attributes[name] = parseStyles(value);
-                    return;
+                    value = parseStyles(value);
+                }
+
+                if(propertiesMap[attr.toLowerCase()] || isCustomAttribute(attr)) {
```

Comment:
It reads a bit confusing if you check for `isCustomAttribute` but call the result a normal attribute